addOns/paramminer/paramminer.gradle.kts

```diff
@@ -9,6 +9,12 @@ zapAddOn {
     }
 }
 
+dependencies {
+    implementation("com.googlecode.java-diff-utils:diffutils:1.3.0")
+    implementation("org.bitbucket.mstrobel:procyon-compilertools:0.5.36")
```

Comment:
I don’t see these used, do you plan to use them or was this just a copy/paste?

Reply:
It was a blind copy paste :)